.github/workflows/perf_micro.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           # sudo taskset 0xef chrt 50.
           # But this makes the process use non-standard, real-time
           # round-robin scheduling mechanism.
-          BENCH_CMD: "taskset 0xfe chrt 50"
+          BENCH_CMD: "taskset 0x01fffffe chrt 50"
```

Comment:
Minor: I would add the comment here about the specific mask ("Hardware machine for benchmarking has a group of isolated CPUs 1-24.") from the commit message.

Reply:
Fixed:

```diff
--- a/.github/workflows/perf_micro.yml
+++ b/.github/workflows/perf_micro.yml
@@ -102,6 +102,8 @@ jobs:
           # sudo taskset 0xef chrt 50.
           # But this makes the process use non-standard, real-time
           # round-robin scheduling mechanism.
+          # Hardware machine for benchmarking has a group of
+          # isolated CPUs 1-24.
           BENCH_CMD: "taskset 0x01fffffe chrt 50"
       - name: Aggregate benchmark results
         run: make -f .test.mk test-perf-aggregate
```